### ISSUE TITLE
Details on reproducing object_autoenc_32

### ISSUE THREAD

**Issue by tonyromarock** [shubhtuls/factored3d#13]
Hi shubtuls,

can you provide more details on how the training setup looked like to produce the pretrained voxel auto-encoder `object_autoenc_32`, which we download before training the factored representation in [training.md](https://github.com/shubhtuls/factored3d/blob/master/docs/training.md#training-factored-3d-prediction-models)? Is there maybe even the script available for creating the auto-encoder on my own?

In the paper you mention using a setup similar to [Girdhar et al.](https://arxiv.org/abs/1603.08637), but you don't go in detail how that training is setup. Do you also construct scenes only consisting of a single object from different azimuth angles and elevation as they did?

I would be very interested in that, since I am doing research on using other bootstrapping measures to improve the vector representation trained in the bottleneck layer.

The documentation of your code here on GitHub for this paper is otherwise excellent.
Thank you in advance!

**Comment by shubhtuls** [comment]
Hi,
Thanks for the interest in the code. We simply trained an auto-encoder with a cross entropy loss.

As a slightly suboptimal way of sharing the code, I'm attaching the two required files (main experiment, and data loader) in this comment - I don't want to incorporate it with the main repo as there are some differences (e.g. different flag library) from the remaining source. Please change the extension to .py instead of .txt (since github does not allow uploading .py).

[object_autoenc.txt](https://github.com/shubhtuls/factored3d/files/2641868/object_autoenc.txt)

[suncg_objects.txt](https://github.com/shubhtuls/factored3d/files/2641871/suncg_objects.txt)
